configure.ac

```diff
@@ -6,6 +6,11 @@ AM_INIT_AUTOMAKE
 
 dnl Checks for programs.
 AC_PROG_CC
+AC_PROG_CC_C99
+if test "x$ac_cv_prog_cc_c99" = "xno"; then
+        AC_MSG_ERROR([Your C compiler does not support C99 standard.])
+fi
```

Comment:
Included in c40f8e2f25131e08e0210626ce017d31a1252ddd also with fixing C99 on Travis.

Reply:
Synced with upstream.